projects/1.19/assets/assemblylinemachines/assemblylinemachines/lang/zh_cn.json

```
@@ -0,0 +1,467 @@
+	"block.assemblylinemachines.corrupt_basalt_empowered_coal_ore": "腐化充能煤矿石",
+	"block.assemblylinemachines.smooth_corrupt_stone": "平滑腐化石头",
+	"block.assemblylinemachines.large_corrupt_stone_bricks": "大型腐化石砖",
+	"block.assemblylinemachines.corrupt_stone_conduit_node": "节点腐化石头",
+	"block.assemblylinemachines.corrupt_stone_conduit": "平行腐化石头",
+	"block.assemblylinemachines.corrupt_stone_conduit_strikethrough_tile": "线纹腐化石砖",
+	"block.assemblylinemachines.corrupt_stone_jewelled_strikethrough_tile": "镶条腐化石砖",
+	"block.assemblylinemachines.corrupt_stone_panel": "饰片腐化石头",
+	"block.assemblylinemachines.corrupt_stone_tile": "腐化石砖",
+	"block.assemblylinemachines.corrupt_stone_pillar": "腐化石柱",
+	
+	"fluid_type.assemblylinemachines.oil":"石油",
+	"fluid_type.assemblylinemachines.condensed_void":"浓缩虚空",
+	"fluid_type.assemblylinemachines.naphtha":"石脑油",
+	"fluid_type.assemblylinemachines.diesel":"柴油",
+	"fluid_type.assemblylinemachines.gasoline":"汽油",
+	"fluid_type.assemblylinemachines.ethane":"乙烷",
+	"fluid_type.assemblylinemachines.propane":"丙烷",
+	"fluid_type.assemblylinemachines.ethylene":"乙烯",
+	"fluid_type.assemblylinemachines.propylene":"丙烯",
+	"fluid_type.assemblylinemachines.liquid_experience":"液态经验",
+	"fluid_type.assemblylinemachines.dark_energy":"暗能量",
+	"fluid_type.assemblylinemachines.liquid_carbon":"液态碳",
+	"fluid_type.assemblylinemachines.glacier_water":"冰川水",
+	
+	"gas.assemblylinemachines.ethane":"乙烷",
+	"gas.assemblylinemachines.propane":"丙烷",
+	"gas.assemblylinemachines.ethylene":"乙烯",
+	"gas.assemblylinemachines.propylene":"丙烯",
+	
+	"itemGroup.assemblylinemachines":"装配线",
+	
+	"enchantment.assemblylinemachines.overclock":"超频",
+	"enchantment.assemblylinemachines.engineers_fury": "工程师之怒",
+	
+	"death.attack.interactor":"%1$s 被撕碎了，凶手是 %2$s 的交互器",
+	"death.attack.interactor.item":"%2$s 的交互器对 %1$s 使用了 %3$s ，效果拔群！",
+	
+	"commands.assemblylinemachines.guide.missing_mod": "Patchouli is not installed.",
+	"commands.assemblylinemachines.guide.success": "Gave a copy of Assembly Lines & You to %1$s player(s).",
+	"commands.assemblylinemachines.chunkfluid.error": "An unexpected error occurred while interfacing with the chunk fluid.",
+	"commands.assemblylinemachines.chunkfluid.get.success": "There is %1$s mB of %2$s in the chunk at %3$s.",
+	"commands.assemblylinemachines.chunkfluid.get.success_empty": "The chunk at %1$s is empty.",
+	"commands.assemblylinemachines.chunkfluid.set.success": "Updated fluid in chunk at %1$s to %2$s mB of %3$s.",
+	"commands.assemblylinemachines.chunkfluid.remove.success": "Removed fluid from chunk at %1$s.",
+	"commands.assemblylinemachines.errors.missing_fluid": "Unknown fluid \"%1$s\"",
+	"commands.assemblylinemachines.errors.flowing_fluid": "The fluid given must be a source fluid.",
+	"commands.assemblylinemachines.makecreative.success": "Successfully made the %1$s at %2$s creative.",
+	"commands.assemblylinemachines.makecreative.error": "Block at %1$s is not a compatible block with this command.",

```

```suggestion
	"commands.assemblylinemachines.guide.missing_mod": "Patchouli is not installed.",
	"commands.assemblylinemachines.guide.success": "Gave a copy of Assembly Lines & You to %1$s player(s).",
	"commands.assemblylinemachines.chunkfluid.error": "An unexpected error occurred while interfacing with the chunk fluid.",
	"commands.assemblylinemachines.chunkfluid.get.success": "There is %1$s mB of %2$s in the chunk at %3$s.",
	"commands.assemblylinemachines.chunkfluid.get.success_empty": "The chunk at %1$s is empty.",
	"commands.assemblylinemachines.chunkfluid.set.success": "Updated fluid in chunk at %1$s to %2$s mB of %3$s.",
	"commands.assemblylinemachines.chunkfluid.remove.success": "Removed fluid from chunk at %1$s.",
	"commands.assemblylinemachines.errors.missing_fluid": "Unknown fluid \"%1$s\"",
	"commands.assemblylinemachines.errors.flowing_fluid": "The fluid given must be a source fluid.",
	"commands.assemblylinemachines.makecreative.success": "Successfully made the %1$s at %2$s creative.",
	"commands.assemblylinemachines.makecreative.error": "Block at %1$s is not a compatible block with this command.",
```
命令交互也是游玩体验的一部分，请尽量翻一下吧，辛苦了（）
#2808有翻译过